examples/config.hocon.sample

```diff
@@ -322,7 +322,27 @@ collector {
       timeLimit = ${?COLLECTOR_STREAMS_BUFFER_TIME_LIMIT}
     }
   }
-
+  # Telemetry sends heartbeat events to external pipeline.
+  # Unless isDisabled parameter set to true, this feature will be enabled. Deleting whole section will not disable it.
+  # Schema URI: iglu:com.snowplowanalytics.oss/oss_context/jsonschema/1-0-1
+  #
+  telemetry {
+      isDisabled = false
```

Comment:
We generally prefer to use a verb instead of passive, see [here](https://github.com/snowplow/enrich/blob/feature/file-sink-rotate/config/config.kinesis.reference.hocon#L197)